### PR TITLE
fix(medusa-plugin-ip-lookup): Remove outdated Typeorm usage

### PR DESCRIPTION
**Pull request by olivermrbl**

### CODE REVIEW

File: packages/medusa-plugin-ip-lookup/src/api/medusa-middleware.js [hunk 1]
```diff
@@ -19,17 +18,20 @@ export default {
         return
       }
 
-      const countryRepo = manager.getCustomRepository(countryRepository)
-      const country = await countryRepo.findOne({
-        where: { iso_2: data.country_code.toLowerCase() },
-      })
+      const lowerCasedCode = data.country_code.toLowerCase()
```
Comment (srindom) [inline packages/medusa-plugin-ip-lookup/src/api/medusa-middleware.js]: Thought: should we normalize within the region service instead?

**Review by srindom — APPROVED** [review]
LGTM w. a small suggestion

File: packages/medusa-plugin-ip-lookup/src/api/medusa-middleware.js (hunk 1 above)
Comment (olivermrbl) [reply packages/medusa-plugin-ip-lookup/src/api/medusa-middleware.js]: yes, for sure - nice catch

Comment (olivermrbl) [reply packages/medusa-plugin-ip-lookup/src/api/medusa-middleware.js:21]: actually, we need both, such that we can attach the lowercased version to the payload

**Comment by olivermrbl** [comment]
Tested locally and works as expected :)

**Pull request merged**